.eslintrc.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  root: true,
+  // This tells ESLint to load the config from the package `eslint-config-grafbase`
+  extends: ['grafbase'],
+  settings: {
+    next: {
+      rootDir: ['apps/*/']
```

Comment:
should be packages?

Reply:
This can be removed, this is a hack for next eslint plugin issue